doc/installation.rst

```diff
@@ -25,6 +25,14 @@ To use it, just run the following commands in your terminal:
   > git clone https://github.com/modoboa/modoboa-installer
   > cd modoboa-installer
   > sudo ./run.py <your domain>
+  
+
+if you get this warning - '/usr/bin/env: ‘python’: No such file or directory'. Run this command first
+
+.. sourcecode:: bash
+
+   > sudo apt-get install python-virtualenv python-pip
```

Comment:
This is true on a debian based system only. Could you indicate this is an example please?

Reply:
How do I do that? --> 'Indicate as an example?'

Reply:
You could add something like: ``If ..., please make sure python is installed on your server. For example on a debian based system:`` ?

Reply:
Done